README.rst

```diff
@@ -1,8 +1,8 @@
 ================================
 Industrial CI *(legacy version)*
 ================================
-.. image:: https://travis-ci.com/ros-industrial/industrial_ci.svg?branch=legacy
-    :target: https://travis-ci.com/ros-industrial/industrial_ci
+.. image:: https://travis-ci.com/ipa320/industrial_ci_mirror/branches.svg?branch=mirror-legacy
```

Comment:
Fixed in https://github.com/ros-industrial/industrial_ci/commit/c1ff67839d513374197f6433d7b964879618e798